seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java

```diff
@@ -281,7 +304,80 @@ private List<String> readSinkData() throws InterruptedException {
                         "c_int",
                         "c_date",
                         "c_timestamp");
-        return getDocsWithTransformTimestamp(source, "st_index2");
+        return getDocsWithTransformTimestamp(source, index);
+    }
+
+    private List<String> readMultiSinkData5(String index) throws InterruptedException {
```

Comment:
what's different between `readMultiSinkData5` and `readMultiSinkData6`?

Reply:
Schema is different. c_string

Reply:
How about like this?
```java
private List<String> readMultiSinkData(String index, List<String> source) throws
```